rootfs/etc/nginx/lua/monitor.lua

```diff
@@ -0,0 +1,50 @@
+local socket = ngx.socket.udp
+local cjson = require('cjson')
+local defer = require('defer')
+local assert = assert
+
+local _M = {}
+
+local function send_data(jsonData)
+  local s = assert(socket())
+  assert(s:setpeername("127.0.0.1", 8000))
+  assert(s:send(jsonData))
+  assert(s:close())
+end
+
+function _M.encode_nginx_stats()
```

Comment:
@ElvinEfendi My initial approach was to test `.call` method. Yesterday I spent a fair bit of the day trying to figure out how to make it work but couldn't find a clean / non-hacky way of testing it. Mostly due to the nature of the module: 1) Generate JSON struct form nginx context variables. 2) Create a timer with send_data callback. 3) Send the JSON struct as a UDP message. The main logic of the module is building the JSON structure (then use the interface of the other modules which are already tested). After pairing with Andrew we decided that this would be the cleanest way of going forward.